fix subtitle in list items

yenda opened the pull request:
arguments were missing in reagent-render args so component wasn't reacting to changes

## Testing

Tested on Android emulator
By default I was always seeing `Available only on Mainnet and Testnet` on ENS username item in profile, now I see the proper subtitle

status: ready

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | ef7f6759 | [#1](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-8888/1/) | 2019-09-02 18:09:55 | ~24 sec | `android` | [:page_facing_up: log](https://ci.status.im/job/status-react/job/prs/job/android/job/PR-8888/1/consoleText) |

Commit:
fix subtitle in list items

arguments were missing in reagent-render args so component wasn't
reacting to changes

Signed-off-by: yenda <eric@status.im>

bitsikka commented:
awesome!!! Thankyou @yenda somewhere in the back of my mind it was nagging me why the chat list subtitle was not re-rendering, when, the same sub components as before was re-used.

This totally explains it. So liberating to have reasons :)

So this means we did not need for subtitle component to subscribe its own subscription 😅 to re-render as needed.

Thankyou also for going easy on me :)

flexsurfer commented:
@bitsikka could you please revert subscriptions then? :)

bitsikka commented:
> @bitsikka could you please revert subscriptions then? :)

of course 👍 
on it

yenda commented:
> Thankyou also for going easy on me :)

No worries that is probably the number one mistake people do with reagent, we've all done it a few times.

churik commented:
Tested on IOS 11.4.1. and Android 8 - on mainnet and ropsten "Register a universal username....", on other networks "Only available on Mainnet and Ropsten"
